### PR TITLE
First draft of IPFS Weekly #25

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft so that we can send it out via TinyLetter (and post on the blog) Tuesday, January 15. 🎉

Thanks!

### CODE REVIEW

**Comment by terichadbourne** [comment]
@renrutnnej I'd love to feature the ProtoSchool launch in this issue. It will be publicly announced on Monday's IPFS Weekly Call, so we'll be safe to share in the Tuesday newsletter. I'll plan to write up a little blurb for you.

**Comment by jennwrites** [comment]
@terichadbourne That’s awesome! I'll make it our top feature.

**Comment by terichadbourne** [comment]
@renrtunnej I don't seem to have access to push to this repo, but I've drafted a potential announcement for you to paste into your draft. If this length doesn't work, just let me know what you need and I can make this shorter for you.

`## Announcing ProtoSchool
This week we're excited to announce the launch of [ProtoSchool](https://proto.school), an educational community that teaches decentralized web protocols and tools through online tutorials and local chapter events.

[Self-guided interactive tutorials](https://proto.school/tutorials) teach new concepts through code challenges completed in the browser, with feedback to help you track your progress. Tutorials are built by community members like you with an interest in helping others learn about IPFS and other decentralized web projects such as IPLD, LibP2P, Multiformats, and Filecoin.

[Local chapter events](https://proto.school/chapters) around the world use ProtoSchool tutorials as curriculum, with mentors available to help participants work through the challenges. ProtoSchool chapters operate independently, but each has its own repo in our org to facilitate chapter discussions and event planning, as well as easy hosting of a chapter website via GitHub Pages. [Start your own chapter today!](https://proto.school/host)`

cc @mikeal

File: published/025-2019-jan-15.md [hunk 1]
```diff
@@ -8,14 +8,22 @@ Want this update in your inbox? [Subscribe to our weekly newsletter!](https://ti
...
+## Announcing ProtoSchool
+
+This week we’re excited to announce the launch of [ProtoSchool](https://proto.school/), an educational community that teaches decentralized web protocols and tools through online tutorials and local chapter events.
+
+[Self-guided interactive tutorials](https://proto.school/#/tutorials) teach new concepts through code challenges completed in the browser, with feedback to help you track your progress. Tutorials are built by community members like you with an interest in helping others learn about IPFS and other decentralized web projects such as IPLD, LibP2P, Multiformats, and Filecoin.
+
+[Local chapter events](https://proto.school/#/chapters) around the world use ProtoSchool tutorials as curriculum, with mentors available to help participants work through the challenges. ProtoSchool chapters operate independently, but each has its own repo in our org to facilitate chapter discussions and event planning, as well as easy hosting of a chapter website via GitHub Pages. [Start your own chapter today!](https://proto.school/#/host)
+
+
```
Comment (terichadbourne) [inline published/025-2019-jan-15.md:20]: @renrutnnej I was able to get some help editing yesterday's launch presentation into a 7.5 minute video that's now available on YouTube.  Would you like to add this at the end of this section? 

`Check out our [launch video](https://youtu.be/G5aiLkHZ7CM) to learn more about how you can get involved with the ProtoSchool community.`

**Comment by terichadbourne** [comment]
@renrutnnej  A video of the ProtoSchool launch preso is now available at https://youtu.be/G5aiLkHZ7CM if this hasn't gone out yet and you'd like to add it.

File: published/025-2019-jan-15.md (hunk 1 above)
Comment (jennwrites) [reply published/025-2019-jan-15.md:20]: @terichadbourne I’ve already sent out the newsletter but I can add this bit to the blog and archival version.

Comment (terichadbourne) [reply published/025-2019-jan-15.md:20]: Sounds perfect. Thanks @renrutnnej !

**Comment by jennwrites** [comment]
IPFS Weekly 25 was published:

+ On the [IPFS blog](https://blog.ipfs.io/63-ipfs-weekly-25/)
+ Sent to 2986 subscribers [via Tinyletter](https://tinyletter.com/ipfsnewsletter/letters/ipfs-weekly-25)

**Pull request merged**